praktikumsplaner-frontend/frontend/src/composables/rules.ts

```diff
@@ -40,13 +47,20 @@ export function useRules() {
             message;
     }
 
+    function regexRule(regex: RegExp, message = "error") {
+        return (value: string | null | undefined) =>
+            (value && regex.test(value)) || message;
+    }
+
```

Comment:
I think either this rule should be specific for the jahrgang or the email rule should be refactored to use this generic regexRule

Reply:
That's true. I removed the email rule, and replaced it with regex rule